client/src/components/steam-user-search-box-wizard.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import classnames from 'classnames';
+import {
+  Button,
+  Form,
+  FormFeedback,
+  FormGroup,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  InputGroupText
+} from 'reactstrap';
+
+export default function (props) {
+  const [search, updateSearch] = useState(props.search || '');
+  const history = useHistory();
+
+  return (
+    <div className={classnames(props.frontpageVersion ? 'shadow' : '', props.className)}>
```

Comment:
There's no frontpage version, is this needed?

Reply:
tbh I just did copy the one from the frontpage will fix it 👍

Reply:
Indeed removed it